tests/console/consoletest_setup.pm

```diff
@@ -31,15 +31,6 @@ sub run {
     save_screenshot;
     check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
 
-    # on s390x we cannot stop the getty, otherwise the iucvconn will break
-    if (!check_var('BACKEND', 's390x')) {
-        systemctl "stop serial-getty\@$testapi::serialdev", ignore_failure => 1;
-        systemctl "disable serial-getty\@$testapi::serialdev";
-        # Mask if is qemu backend as use serial in remote installations e.g. during reboot
-        systemctl "mask serial-getty\@$testapi::serialdev" if check_var('BACKEND', 'qemu');
-    }
-
-    check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
```

Comment:
@foursixnine, isn't this code a duplicate of disable_**serial_getty()**?

Reply:
@SergioAtSUSE I think yes to some extent, but in the end there's a console being selected... which is clearly a different behaviour. Perhaps call it before L32?